src/twisted/conch/test/test_manhole_ssh.py

```diff
@@ -0,0 +1,21 @@
+from hamcrest import assert_that
+
+from twisted.cred.portal import IRealm
+from twisted.trial import unittest
+from twisted.trial.test.matchers import provides
+
+try:
+    from twisted.conch.manhole_ssh import TerminalRealm
+except ImportError:
+    ssh = False
+else:
+    ssh = True
+
+
+class TerminalRealmTests(unittest.TestCase):
+    if not ssh:
+        skip = "cannot import t.conch.manhole_ssh (cryptography requirements missing?)"
```

Comment:
minor comment.

mabye we can remove the last if :)

```suggestion
except ImportError:
    skipNoConch = 'No t.conch.manhole_ssh available.
else:
    skipNoConch = None


class TerminalRealmTests(unittest.TestCase):
    skip = skipNoConch
```